lib/bank/savings_account.rb

```diff
@@ -0,0 +1,30 @@
+module Bank
+  class SavingsAccount < Account
+
+    def initialize(id, initial_balance)
+      raise ArgumentError, "Balance must be a minimum of $10." if initial_balance < 10
+
+      @id = id
+      @balance = initial_balance.to_i
+      @minimum_balance = 12
+    end
+
+    def withdraw(amount)
+      super
+      # there was not an error with the withdrawal amount
+      if !@is_withdrawal_error
+        # subtract the $2 fee from balance
+        return @balance - 2
```

Comment:
Same as CheckingAccount, this returns what looks like an updated balance, but the fee is not actually accessed on the transaction.